boards/cats_rev1Pro/src/config/globals.c

```diff
@@ -119,13 +119,13 @@ fifo_t usb_input_fifo;
 fifo_t usb_output_fifo;
 
 uint8_t usb_fifo_out_buffer[256];
-uint8_t usb_fifo_in_buffer[64];
+uint8_t usb_fifo_in_buffer[256];
```

Comment:
can we have a define for buffer sizes? otherwise we have to change it in more places